aestream/__init__.py

```diff
@@ -11,11 +11,11 @@
     logging.debug("Failed to import Torch: AEStream is running in Numpy mode")
     del logging
 
-from .aestream_ext import FileInput, UDPInput
+from aestream.input import FileInput, UDPInput
 
 modules = []
 try:
```

Comment:
Are these changes effectively the same as before? If not maybe it's worth solidifying how the API will be implemented first and reviewing the structure of the modules.